bundler/spec/commands/newgem_spec.rb

```diff
@@ -181,6 +181,10 @@ def bundle_exec_standardrb
       it "generates a default .rubocop.yml" do
         expect(bundled_app("#{gem_name}/.rubocop.yml")).to exist
       end
+
+      it "contained .rubocop.yml into ignore list" do
+        expect(bundled_app("#{gem_name}/#{gem_name}.gemspec").read).to include("appveyor .rubocop.yml Gemfile")
```

Comment:
Maybe use

```suggestion
        expect(bundled_app("#{gem_name}/#{gem_name}.gemspec").read).to include(" .rubocop.yml ")
```

instead, so that if we for example remove `appveyor` from this list, these specs don't need updates?